lib/iex/errors/symbol_not_found_error.rb

```diff
@@ -1,14 +1,27 @@
 module IEX
   module Errors
-    class SymbolNotFoundError < StandardError
-      attr_reader :symbol
+    class IexError < StandardError
```

Comment:
added an Error superclass to allow for other projects to:
``` Ruby
rescue IEX::Errors::IexError
```
instead of having to:
``` Ruby
rescue IEX::Errors::SymbolNotFoundError, IEX::Errors::SearchNotFoundError, ... # as more errors are added
```

Reply:
tbh, I hadn't realized there were other errors defined (I thought I was creating the second one). If I had, I probably would have put this into a separate PR. sorry for the confusion!